dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/PIAntiWindup.mo

```diff
@@ -0,0 +1,75 @@
+within Dynawo.NonElectrical.Blocks.Continuous;
+
+/*
+* Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+block PIAntiWindup
```

Comment:
I think the name is quite clear but we never have too much comments.

Reply:
Fixed